bench/bench-shell-bindings.js

```diff
@@ -3,21 +3,28 @@ if (typeof load !== "function") {
   var vm = require("vm");
   load = function(file) {
     var src = fs.readFileSync(file, "utf8");
-    vm.runInThisContext(src);
+    vm.runInThisContext(src, { filename: file });
   };
 }
 
 if (typeof print !== "function") {
-  print = console.log.bind(console);
+  print = function (x = "") {
+    console.log(`${x}`);
```

Comment:
Not important but I thought `console.log` was self-bound already.

Reply:
I know it differed across engines at one point in time, unsure if it still does or not.

In fact, the shell version of the benchmarks doesn't work anymore (added too many options that are only exposed in the Webpage), so I should just remove it.